### ISSUE TITLE
kindly adding some documentations on t5-v1_1-base""

### ISSUE THREAD

**Issue by dorooddorood606** [huggingface/transformers#12990]
## Environment info

- `transformers` version:
- Platform:
- Python version:
- PyTorch version (GPU?):
- Tensorflow version (GPU?):
- Using GPU in script?:
- Using distributed or parallel set-up in script?:

### Who can help

Documentation: @sgugger
Hi
Could you kindly add some documentations on "t5-v1_1-base"? I tested one code with t5-base and t5-v1 version, for t5-v1 I got memory issue, this seems to me the model size is different and larger, also fast tokenizer for this model does not work, could you kindly add a documentation on these differences?

thanks a lot.

**Comment by sgugger** [comment]
There is no model named `"t5-v1_1-base"` so I'm not sure what you mean.

**Comment by NielsRogge** [comment]
Yes there is, `google/t5-v1_1-base`. Normally, t5_v1_1 and regular t5 aren't that different. From the model card:

> Version 1.1
T5 Version 1.1 includes the following improvements compared to the original T5 model- GEGLU activation in feed-forward hidden layer, rather than ReLU - see here.
Dropout was turned off in pre-training (quality win). Dropout should be re-enabled during fine-tuning.
Pre-trained on C4 only without mixing in the downstream tasks.
no parameter sharing between embedding and classifier layer
"xl" and "xxl" replace "3B" and "11B". The model shapes are a bit different - larger d_model and smaller num_heads and d_ff.
Note: T5 Version 1.1 was only pre-trained on C4 excluding any supervised training. Therefore, this model has to be fine-tuned before it is useable on a downstream task. Pretraining Dataset: C4

So for the base-sized model, normally the memory requirements are the same as t5-base.

**Comment by NielsRogge** [comment]
I have added documentation, see #13240. Therefore, closing.
